### ISSUE TITLE
Clean up "monetizing your newsletter" doc

### ISSUE THREAD

**Issue by madelinezday** [buttondown-email/roadmap#1441]
Added some high level instructions here:

https://www.loom.com/share/b8632b94940b45c79f1aa2dd4dd1bdf8

**Comment by madelinezday** [comment]
Here's my work-in-progress draft! I'm still smoothing out the language a bit, but just wanted to flag a few of the questions in the comments! 

https://docs.google.com/document/d/1CQRqSvZltSnVRY6lbHV-cvJ_gWslMNbmdJGygmfsIRY/edit?usp=sharing

**Comment by jmduke** [comment]
Thanks for the flag: answered most of 'em! 

I'll also set up a 'test' Stripe account for you just so you have access to what it looks like for gifs + screenshots going forward.

**Comment by madelinezday** [comment]
Here's the formatted doc with your edits ! However, I don't think I'm able to see the "Gift Subscriptions" interface without my Stripe account set up. Perhaps we could hop on a quick call to discuss how the interface works, and then I can add that in to the "Gift Subscriptions" section?

I also wanted to flag that I moved the Stripe integration section over to its own guide, which is in the "In Review" section of this board! If you'd like me to add that info back in to this doc, I'm happy to do that as well, but I think that this shorter doc is a little less daunting! I also added brackets throughout the doc to indicate where we can link to the Stripe integration guide. 

https://github.com/madelinezday/buttondown/blob/main/guides/monetizing-your-newsletter.md
